opsimulator/opsimulator.go

```diff
@@ -81,6 +87,35 @@ func (opSim *OpSimulator) Start(ctx context.Context) error {
 		opSim.port = uint64(port)
 	}
 
+	bgTasksCtx, bgTasksCancel := context.WithCancel(ctx)
+
+	opSim.bgTasksCancel = bgTasksCancel
+	// Not handling or bubbling up errors now
+	opSim.bgTasks = tasks.Group{}
+
+	// Relay deposit tx from L1 to L2
+	opSim.bgTasks.Go(func() error {
+		depositTxCh := make(chan *types.DepositTx)
+		sub, err := SubscribeDepositTx(context.Background(), opSim.l1Chain, opSim.L2Config.L1DeploymentAddresses.OptimismPortalProxy, depositTxCh)
+
+		if err != nil {
+			return fmt.Errorf("failed to subscribe to deposit tx: %w", err)
+		}
+
+		for {
+			select {
+			case dep := <-depositTxCh:
```

Comment:
nit: debug log that a deposit transaction was received.

Reply:
good point! fixed